tests/e2e/kubetest2-kops/deployer/up.go

```diff
@@ -178,6 +182,21 @@ func (d *deployer) createCluster(zones []string, adminAccess string, yes bool) e
 		args = appendIfUnset(args, "--gce-service-account", "default")
 		args = appendIfUnset(args, "--networking", "kubenet")
 
+		// kubeapi server config https://github.com/kubernetes/kubernetes/blob/622509830c1038535e539f7d364f5cd7c3b38791/cluster/gce/gci/configure-kubeapiserver.sh#L106
+		for _, v := range strings.Split(d.KubeAPIServerAdmissionPlugins, ",") {
+			args = append(args, "--set", fmt.Sprintf("spec.kubeAPIServer.enableAdmissionPlugins=%s", v))
+		}
+		if strings.Contains(d.KubernetesFeatureGates, "AllAlpha") {
+			d.KubeAPIServerRuntimeConfig = "api/all=true"
```

Comment:
```suggestion
			d.KubeAPIServerRuntimeConfig = "api/all=true," + d.KubeAPIServerRuntimeConfig
```

I initially wrote this as I'm currently suggesting, but it is broken(only the final arg is actually stored). `api/all=true` enables all APIs and I was able to successfully launch the following test run

```
kubetest2 kops --up --down --build --stage-location gs://kubernetes-staging-a1db298f53 --cluster-name amd64.k8s.local --cloud-provider=gce   --admin-access="0.0.0.0/0"   --gcp-project=borg-env   --create-args="--image='ubuntu-os-cloud/ubuntu-2204-jammy-v20230908' --channel=alpha --zones=us-central1-a --node-count=2" --kops-binary-path $GOPATH/src/k8s.io/kops/.build/dist/linux/amd64/kops --kubernetes-version=https://storage.googleapis.com/k8s-release-dev/ci/latest.txt --kubernetes-feature-gates=+AllAlpha,-InTreePluginGCEUnregister,+DisableCloudProviders,+DisableKubeletCloudCredentialProviders --test=kops --  --test-args="-test.timeout=180m -num-nodes=3 --master-os-distro=gci --node-os-distro=gci"   --test-package-bucket=k8s-release-dev   --test-package-dir=ci   --test-package-marker=latest.txt   --focus-regex="\[Feature:(AdmissionWebhookMatchConditions|InPlacePodVerticalScaling|SidecarContainers|StorageVersionAPI|PodPreset|StatefulSetAutoDeletePVC)\]|Networking"   --skip-regex="\[Feature:(SCTPConnectivity|Volumes|Networking-Performance)\]|IPv6|csi-hostpath-v0"   --parallel=1
```

Reply:
should we update the list of admissions plugins?